remoulade/middleware/middleware.py

```diff
@@ -39,59 +38,62 @@ class Middleware:
     default_before = None  # type: Type[Middleware]
     default_after = None  # type: Type[Middleware]
 
+    def get_option_value(self, option_name, broker, message, *, default=None):
```

Comment:
I don't really like this name,  just `get_option` ? What do you think ?

Reply:
Also I would prefer this signature, `def get_option_value(self, option_name, *, broker, message, default=None)` to make things more clear.